cognite/client/data_classes/aggregations.py

```diff
@@ -149,21 +149,19 @@ def _load(cls, resource: dict[str, Any], cognite_client: CogniteClient | None =
         return cast(Self, deserialized)
 
     def dump(self, camel_case: bool = True) -> dict[str, Any]:
-        output = {"aggregate": self._aggregate, "property": self.property}
-        if camel_case:
-            output = convert_all_keys_recursive(output)
-        return output
+        return {"aggregate": self._aggregate, "property": self.property}
```

Comment:
This change isn't described by your PR description. Is it accidental, or deliberately but undocumented?

Reply:
Nice to see you remove that function 👌